client/components/Snackbars.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert from '@material-ui/lab/Alert';
+import snackbarActions from '../store/actions/snackbars';
+
+const Snackbars = ({ snackbars, hideSnackbar, removeSnackbar }) => {
+  const handleHideSnackbar = (event, reason) => {
+    if (reason !== 'clickaway') hideSnackbar();
```

Comment:
so what happens for a clickaway? Nothing?

Reply:
Yeah, I'd find it weird if randomly clicking anywhere on the page dismisses the snackbar that has an explicit close button :-)